### PR TITLE
[stable/openebs] Upgrade to openebs release 0.6

### PR DESCRIPTION
**Pull request by kmova**
**What this PR does / why we need it**:

This PR includes the following changes:
- Update the image tags to 0.6
- Add a new deployment introduced in 0.6 for supporting snapshots
- Identation fix with templates
- Updated README and Chart.yaml

Signed-off-by: kmova <kiran.mova@openebs.io>

### CODE REVIEW

**Comment by kmova** [comment]
@unguiculus @mgoodness @seanknox - can you please help reviewing this PR.

File: stable/openebs/templates/deployment-maya-snapshot-operator.yaml [hunk 1]
```diff
@@ -0,0 +1,91 @@
...
+{{- if .Values.snapshotOperator.nodeSelector }}
+      nodeSelector:
+{{ toYaml .Values.snapshotOperator.nodeSelector | indent 8 }}
+{{- end }}
+{{- if .Values.snapshotOperator.tolerations }}
+      tolerations:
+{{ toYaml .Values.snapshotOperator.tolerations | indent 8 }}
+{{- end }}
+{{- if .Values.snapshotOperator.affinity }}
+      tolerations:
```
Comment (unguiculus) [inline stable/openebs/templates/deployment-maya-snapshot-operator.yaml]: affinity

**Review by unguiculus — CHANGES_REQUESTED** [review]

File: stable/openebs/values.yaml [hunk 2]
```diff
@@ -26,19 +26,39 @@ apiserver:
...
+  controller:
+    image: "openebs/snapshot-controller"
+    imageTag: "0.6.0"
+  provisioner:
+    image: "openebs/snapshot-provisioner"
+    imageTag: "0.6.0"
+  replicas: 1
+  upgradeStrategy: "Recreate"
+  nodeSelector: {}
+  tolerations: {}
```
Comment (unguiculus) [inline stable/openebs/values.yaml]: `tolerations: []`

**Review by unguiculus — CHANGES_REQUESTED** [review]

File: stable/openebs/templates/deployment-maya-snapshot-operator.yaml (hunk 1 above)
Comment (kmova) [reply stable/openebs/templates/deployment-maya-snapshot-operator.yaml]: thank you! fixed it.

File: stable/openebs/values.yaml (hunk 2 above)
Comment (kmova) [reply stable/openebs/values.yaml]: thank you! fixed it.

**Review by unguiculus — APPROVED** [review]

**Pull request merged**